pkg/synthetictests/alerts.go

```diff
@@ -32,5 +90,121 @@ func testAlerts(events monitorapi.Intervals, restConfig *rest.Config, duration t
 		ret = append(ret, junit...)
 	}
 
+	// Run the backstop catch all for all other alerts:
+	ret = append(ret, runBackstopTest(allowancesFunc, featureSet, events, alertTests)...)
+
+	return ret
+}
+
+// runBackstopTest will process the intervals for any alerts which do not have their own explicit test,
+// and look for any pending/firing intervals that are not within sufficient range.
+func runBackstopTest(
+	allowancesFunc alerts.AllowedAlertsFunc,
+	featureSet configv1.FeatureSet,
+	alertIntervals monitorapi.Intervals,
+	alertTests []allowedalerts.AlertTest) []*junitapi.JUnitTestCase {
```

Comment:
The core of the new invariant backstop test that checks all alerts that don't have their own explicit test. Above function is just for the logic around a live cluster, this is the one we can use to run locally and pass in our inputs.